### ISSUE TITLE
Add a "reload for offline" button in the internal web browser view

### ISSUE THREAD

**Issue by axkr** [JonasCz/save-for-offline#6]
Add a "reload for offline" button in the internal web browser view

**Comment by JonasCz** [comment]
Will do it.

**Comment by JonasCz** [comment]
I have lots of other things to do, and this is not on my priority list. It would require changes in various places, so it will be a while before I implement this. By this you mean to 'refresh' the saved version of the page?

**Comment by ghost** [comment]
Automatic reloading at certain intervals would be handy too. Such as every 6 hours for a weather forecast page.

**Comment by albertincx** [comment]
@JonasCz  How to talk with you?

**Comment by JonasCz** [comment]
I'm very busy doing other things, and I can't implement this feature at the moment, sorry @albertincx 

If you have issues relating to this app, the issues here on GitHub is the place to report them, otherwise I'm not really available for anything else. I probably won't get round to fixing any issues or adding features for quite some time.

Pull requests are welcome though, and I'll try to get round to them.

**Comment by albertincx** [comment]
@JonasCz how to update it in the app store ? I want to add synchronization with other devices , and a few magical things.

**Comment by JonasCz** [comment]
@albertincx  Someone else took my source and published it in the android playstore, which means that I can't update it. It's a very old version, unfortunately, and the person who uploaded it to Google Play is not updating it.

I may eventually create a google play developer account and publish the new version, but not at the moment. Feel free to take the app / source and upload it to the playstore yourself though, if you want to do so, but I can't at the moment.

**Comment by albertincx** [comment]
@JonasCz allow me to modify the code and upload to the store under a new brand ?

**Comment by JonasCz** [comment]
@albertincx Yes, feel free to modify the code and do whatever you want with it (It's GPL licensed, so you can do whatever is allowed under that licence). I'll give you commit access to this repository, so you can modify the code here without using pull requests. Feel free to also change the name of the app in this repository, so you don't have to maintain a fork of the app. You'll need to change the package name too in order to upload it to the playstore.
